From 94187fb3463f2f7ef67d067b5e98f5b91b7f04d8 Mon Sep 17 00:00:00 2001
From: Tomas Zak <tomas.zak@turris.com>
Date: Thu, 16 Nov 2023 10:44:52 +0100
Subject: [PATCH] packages: python-six python-pytz: adding hostbuild option

Added hostbuild option for packages python-six and python-pytz.
We need these two packages as hostbuild for our turri-timezone package.
---
 lang/python/python-pytz/Makefile | 8 ++++++++
 lang/python/python-six/Makefile  | 8 ++++++++
 2 files changed, 16 insertions(+)

diff --git a/lang/python/python-pytz/Makefile b/lang/python/python-pytz/Makefile
index 62c1b3a2c..f445d3a31 100644
--- a/lang/python/python-pytz/Makefile
+++ b/lang/python/python-pytz/Makefile
@@ -18,9 +18,16 @@ PKG_MAINTAINER:=Alexandru Ardelean <ardeleanalex@gmail.com>
 PKG_LICENSE:=MIT
 PKG_LICENSE_FILES:=LICENSE.txt
 
+HOST_BUILD_DEPENDS:=python3/host \
+                    python-build/host \
+                    python-installer/host \
+                    python-wheel/host
+
 include ../pypi.mk
 include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/host-build.mk
 include ../python3-package.mk
+include ../python3-host-build.mk
 
 define Package/python3-pytz
   SUBMENU:=Python
@@ -38,3 +45,4 @@ endef
 $(eval $(call Py3Package,python3-pytz))
 $(eval $(call BuildPackage,python3-pytz))
 $(eval $(call BuildPackage,python3-pytz-src))
+$(eval $(call HostBuild))
diff --git a/lang/python/python-six/Makefile b/lang/python/python-six/Makefile
index 6712baa3b..08b436e90 100644
--- a/lang/python/python-six/Makefile
+++ b/lang/python/python-six/Makefile
@@ -18,9 +18,16 @@ PKG_LICENSE:=MIT
 PKG_LICENSE_FILES:=LICENSE
 PKG_MAINTAINER:=Jeffery To <jeffery.to@gmail.com>
 
+HOST_BUILD_DEPENDS:=python3/host \
+                    python-build/host \
+                    python-installer/host \
+                    python-wheel/host
+
 include ../pypi.mk
 include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/host-build.mk
 include ../python3-package.mk
+include ../python3-host-build.mk
 
 define Package/python3-six
   SECTION:=lang
@@ -41,3 +48,4 @@ endef
 $(eval $(call Py3Package,python3-six))
 $(eval $(call BuildPackage,python3-six))
 $(eval $(call BuildPackage,python3-six-src))
+$(eval $(call HostBuild))
-- 
2.42.1

